lib/firefox/webdriver/firefoxDelegate.js

```diff
@@ -52,6 +52,7 @@ class FirefoxDelegate {
       let bufferSize = this.firefoxConfig.geckoProfilerParams.bufferSize;
 
       // Firefox 69 and above has a slightly different API for StartProfiler
+      // See https://searchfox.org/mozilla-central/source/tools/profiler/gecko/nsIProfiler.idl#38
```

Comment:
This should probably be a permalink if we want to reference a specific version of the IDL

Reply:
It's mostly to get to the idl definition. Maybe we can use, so it's future proof even if it moves around:

https://searchfox.org/mozilla-central/search?q=symbol:%23StartProfiler%2C_ZN11nsIProfiler13StartProfilerEjdRK8nsTArrayI9nsTStringIcEES5_md&redirect=false